### PR TITLE
[ADD] Add new module stock_picking_partner_language

### PR DESCRIPTION
**Pull request by hurrinico**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Isn't better to pass this as an Odoo/OCB bug/PR?

**Comment by hurrinico** [comment]
@pedrobaeza i have thought about it but i have noticed this https://github.com/odoo/odoo/pull/3686#issuecomment-137048248 . So i have thought that this PR could be better.

**Comment by pedrobaeza** [comment]
Uhm, I see. Then this module can have sense. I would rename it to shorten a bit _stock_picking_partner_lang_.

File: stock_picking_partner_language/README.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Stock Picking Partner Language
```
Comment (pedrobaeza) [inline stock_picking_partner_language/README.rst]: Put upper line for highlighting the title

File: stock_picking_partner_language/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Stock Picking Partner Language',
+    'version': '1.0',
```
Comment (pedrobaeza) [inline stock_picking_partner_language/__openerp__.py]: Use 8.0.1.0.0

File: stock_picking_partner_language/picking_report.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <report
```
Comment (pedrobaeza) [inline stock_picking_partner_language/picking_report.xml]: Is it needed to overwrite full report definition? I think you can use normal XML record definition for overwriting only affected fields (name and file)

Comment (hurrinico) [inline stock_picking_partner_language/picking_report.xml]: @pedrobaeza in our expirience, customer might need 2 reports: one with the user language for "internal use" and one with the partner language.
What do you think about it?

Comment (pedrobaeza) [inline stock_picking_partner_language/picking_report.xml]: Ah, OK, I see. You're not overwriting but creating another one. Totally agree then.

**Comment by hurrinico** [comment]
@pedrobaeza i've done the requested changes

File: stock_picking_partner_language/README.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (pedrobaeza) [inline stock_picking_partner_language/README.rst]: There's missing the target line (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)

**Comment by pedrobaeza** [comment]
Don't you consider the module renaming?

File: stock_picking_partner_language/README.rst [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+==============================
+Stock Picking Partner Language
+==============================
+
+This module allows to print pickings in partner language.
+
+
```
Comment (tafaRU) [inline stock_picking_partner_language/README.rst]: Please remove one empty line

File: stock_picking_partner_language/README.rst [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/stock-logistics-reporting/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/stock-logistics-reporting/issues/new?body=module:%stock_picking_partner_language%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
```
Comment (tafaRU) [inline stock_picking_partner_language/README.rst]: Same as above.

File: stock_picking_partner_language/__init__.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Agile Business Group sagl
```
Comment (tafaRU) [inline stock_picking_partner_language/__init__.py]: Please update copyright year.

File: README.md [hunk 8]
```diff
@@ -4,4 +4,6 @@
 Odoo Stock Logistics Reporting
 ==============================
 
-Reports for the Odoo Logistics modules. 
+Reports for the Odoo Logistics modules.
+
+ - Stock Picking Partner Language: Add possibility to print picking in partner lenguage
```
Comment (tafaRU) [inline README.md]: s/Add/adds
s/lenguage/language

File: stock_picking_partner_language/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Stock Picking Partner Language',
+    'version': '8.0.1.0.0',
+    'category': 'Warehouse Management',
+    'summary': 'Add report for stock picking in partner language',
```
Comment (tafaRU) [inline stock_picking_partner_language/__openerp__.py]: s/Add/Adds

File: stock_picking_partner_language/picking_report.xml [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+            string="Picking In Partner Lenguage" 
+            id="action_report_partner_picking" 
+            model="stock.picking" 
+            report_type="qweb-pdf"
+            name="stock_picking_partner_language.report_picking_partner_language" 
+            file="stock_picking_partner_language.report_picking_partner_language"
+        />
+
+    </data>
+</openerp>
```
Comment (tafaRU) [inline stock_picking_partner_language/picking_report.xml]: No newline at end file

File: stock_picking_partner_language/views/stock_picking_partner_language.xml [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+<data>
+<template id="report_picking_partner_language">
+    <t t-call="report.html_container">
+        <t t-foreach="doc_ids" t-as="doc_id">
+            <t t-raw="translate_doc(doc_id, doc_model, 'partner_id.lang', 'stock.report_picking')"/>
+        </t>
+    </t>
+</template>
+</data>
+</openerp>
```
Comment (tafaRU) [inline stock_picking_partner_language/views/stock_picking_partner_language.xml]: Same as above

File: stock_picking_partner_language/picking_report.xml [hunk 12]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <report 
+            string="Picking In Partner Lenguage"
```
Comment (tafaRU) [inline stock_picking_partner_language/picking_report.xml]: s/Lenguage/Language

**Comment by hurrinico** [comment]
@pedrobaeza @tafaRU i have made the requested fix

**Comment by pedrobaeza** [comment]
Thanks! When CIs goes green, :+1:

File: stock_picking_partner_lang/README.rst [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+
+This module allows to print pickings in partner language.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/stock-logistics-reporting/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/stock-logistics-reporting/issues/new?body=module:%stock_picking_partner_language%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (tafaRU) [inline stock_picking_partner_lang/README.rst]: Please fix the module name: use the new one.

**Comment by hurrinico** [comment]
@tafaRU  done

**Comment by tafaRU** [comment]
@hurrinico, the new issue link seems broken. I tested it from https://github.com/hurrinico/stock-logistics-reporting/tree/add_stock_picking_partner_language/stock_picking_partner_lang
Please check it.

**Comment by hurrinico** [comment]
@tafaRU it works now

**Comment by tafaRU** [comment]
LGTM :+1:

**Comment by pedrobaeza** [comment]
The idea behind version on issue is to reflect Odoo version, not module version, because people is not going to reflect the correct module version for sure, but at least we know in which "branch", so you should put 8.0.

Please also squash all commits in one, as this is a new module and there's no need for the commit history for the reviewed changes. When you do it, I'll merge.

**Comment by hurrinico** [comment]
Ok @pedrobaeza , Thank you.
I have squash all commits in one

File: README.md [hunk 14]
```diff
@@ -4,4 +4,6 @@
 Odoo Stock Logistics Reporting
 ==============================
 
-Reports for the Odoo Logistics modules. 
+Reports for the Odoo Logistics modules.
+
+ - Stock Picking Partner Language: adds possibility to print picking in partner language
```
Comment (tafaRU) [inline README.md]: Please change to 'Stock Picking Partner Lang' according to the new module name.
Do the same thing in the message commit :wink:

**Comment by hurrinico** [comment]
@tafaRU done

**Comment by tafaRU** [comment]
Thanks, @hurrinico!

**Comment by eLBati** [comment]
Thanks :+1:

**Comment by tafaRU** [comment]
Coverage state seems related to https://github.com/OCA/maintainer-quality-tools/issues/247 I merge then.

**Pull request merged**